coffee_core/src/coffee.rs

```diff
@@ -257,12 +283,13 @@ impl PluginManager for CoffeeManager {
         if let Some(index) = plugins.iter().position(|x| x.name() == plugin) {
             let plugin = plugins[index].clone();
             let exec_path = plugin.exec_path.clone();
+            fs::remove_dir_all(plugin.root_path.clone()).await?;
```

Comment:
You should check if the root path is not the repository path, in this way you are corrupting the repository for the current user of coffee (just mee?)

So here you need to check if the repository path of the plugin is == to the plugin.root path (in some way)

Reply:
Do you mean to check if `plugin.root_path` is not in `.coffee/repositories` so not to remove the cloned version?

Reply:
if the root path is equal to the `{repository_root_path}/{plugin_name}` this means that you are inside the repository and the plugin is not installed with the new procedure. So you are deleting the repository and corrupting it

Reply:
>Do you mean to check if plugin.root_path is not in .coffee/repositories so not to remove the cloned version?

This is true for new plugins but what happens if you have coffee that already indexing and install a plugin with the current master version?

If I ran you the `coffee remove` with my current setup coffee delete the directory inside the repository because you are not thinking to be retro compatible with older version of coffee